### PR TITLE
Audio messages

### LINKED ISSUE #83: Broaden the multimedia support
Voice massages and music files has to be playable on click like the video files currently.

Ideally videos (fullscreen mode) and audios also to be included in the gallery with the playback/stop switch on click. When playback activated (and not stopped actively by the user) after the media is finished, switch to the next page, and (if it is playable) playback trigger automatically - like in a playlist.

### PR DESCRIPTION
**Pull request by almirsarajcic**
Resolves #83.

- [x] Support for audio files
- [x] Improved player
- [x] Push to talk button

Screenshot 2023-01-18 at 11 56 23

I couldn't get the waveform to work for the exported messages because Peaks.js can't send an HTTP request to the audio file to analyze it due to CORS.
Screenshot 2023-01-18 at 11 52 17
Therefore, I display the native player for the exported messages.

There's an option to analyze the file on the server and pass the waveform data to the client, but not sure if it's viable for RPi.

### CODE REVIEW

File: test/chat/dialog_test.exs [hunk 1]
```diff
@@ -110,7 +110,7 @@ defmodule Chat.Dialogs.DialogTest do
 
     [bob_version] = Dialogs.read(dialog, bob)
 
-    assert :file = bob_version.type
+    assert :audio = bob_version.type
```
Comment (sergey-lukianov) [inline test/chat/dialog_test.exs:113]: why `:file` becomes `:audio`?

File: test/chat/room_test.exs [hunk 2]
```diff
@@ -56,7 +56,7 @@ defmodule Chat.Rooms.RoomTest do
     assert [
              %Rooms.PlainMessage{content: ^message, type: :text, author_hash: ^alice_hash},
              %Rooms.PlainMessage{type: :memo},
-             %Rooms.PlainMessage{type: :file},
+             %Rooms.PlainMessage{type: :audio},
```
Comment (sergey-lukianov) [inline test/chat/room_test.exs:59]: same here

**Comment by sergey-lukianov** [comment]
> I couldn't get the waveform to work for the exported messages because Peaks.js can't send an HTTP request to the audio file to analyze it due to CORS.

The RPi will no have the connection to send anything. We have no internet connection.

File: test/chat/dialog_test.exs (hunk 1 above)
Comment (almirsarajcic) [reply test/chat/dialog_test.exs:113]: Before, we didn't support audio files so it was recognized as `:file`, but now we do and the test uses a fake file with `audio/mp3` mime type:
https://github.com/Buckitup-chat/chat/blob/6595e1f5110f5fcae49d0a4a9eceb2189c1a32a2/test/support/fake_data.ex#L5-L11

**Review by sergey-lukianov — APPROVED** [review]

**Pull request merged**